scm-ui/ui-components/src/MarkdownView.tsx

```diff
@@ -132,7 +140,7 @@ class MarkdownView extends React.Component<Props> {
             className="content is-word-break"
             skipHtml={skipHtml}
             escapeHtml={skipHtml}
-            source={content}
+            source={this.enrichCommitMentionsWithInternalLinks(content)}
```

Comment:
I don't think that this is safe. This would match texts which are already links. It would be better, if we just search in text blocks.